config/envs/default.py

```diff
@@ -15,3 +15,9 @@ class DefaultConfig(object):
         FLASK_ROOT / "tests" / "controller_tests" / "resources" / "Post_transform_EXAMPLE_data.xlsx"
     )
     ENABLE_PROFILER = os.getenv("ENABLE_PROFILER")
+
+    AWS_REGION = os.getenv("AWS_REGION")
+    AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID")
+    AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
+    AWS_ENDPOINT_OVERRIDE = os.getenv("AWS_ENDPOINT_OVERRIDE")
+    AWS_S3_BUCKET_FAILED_FILES = os.getenv("AWS_S3_BUCKET_FAILED_FILES")
```

Comment:
A lot of these won't be used on AWS, can we move them into `development.py` env config instead?